notebooks/lesson09-overfitting.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<p>wrap this in a function for reuse? ditto for the other plots / fitting curves?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>